.github/workflows/build.yml

```diff
@@ -50,3 +50,22 @@ jobs:
       - name: Doctest
         run: |
           make doctest -C docs/
+      - name: Send mail
+        # Notify when cron job fails
+        if: (github.event_name == 'schedule' && failure())
```

Comment:
Its weird I can't check failure by job.status == 'failure'

Reply:
Maybe a more pointed question if anyone knows is I don't understand when I have to use something like  ${{ github.job }} instead of just github.event_name